test-poc/showcase/src/test/java/org/keycloak/test/showcase/UserTest.java

```diff
@@ -0,0 +1,441 @@
+package org.keycloak.test.showcase;
+
+import jakarta.ws.rs.core.Response;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.keycloak.common.util.Base64;
+import org.keycloak.models.UserModel;
+import org.keycloak.models.credential.PasswordCredentialModel;
+import org.keycloak.models.utils.ModelToRepresentation;
+import org.keycloak.models.utils.StripSecretsUtils;
+import org.keycloak.representations.idm.CredentialRepresentation;
+import org.keycloak.representations.idm.ErrorRepresentation;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.keycloak.test.framework.annotations.InjectOAuthClient;
+import org.keycloak.test.framework.annotations.InjectPage;
+import org.keycloak.test.framework.annotations.InjectRealm;
+import org.keycloak.test.framework.annotations.InjectWebDriver;
+import org.keycloak.test.framework.annotations.KeycloakIntegrationTest;
+import org.keycloak.test.framework.oauth.OAuthClient;
+import org.keycloak.test.framework.page.LoginPage;
+import org.keycloak.test.framework.realm.ManagedRealm;
+import org.keycloak.test.framework.util.ApiUtil;
+import org.keycloak.util.JsonSerialization;
+import org.openqa.selenium.WebDriver;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@KeycloakIntegrationTest
+public class UserTest {
+
+    @InjectRealm
+    ManagedRealm realm;
+
+    @InjectOAuthClient
+    OAuthClient oAuthClient;
+
+    @InjectWebDriver
+    WebDriver webDriver;
+
+    @InjectPage
+    LoginPage loginPage;
```

Comment:
Would remove these as they are not currently used, and honestly using these should not be done in a test that tests admin endpoints. The commented out test later should really be separated out into a separate place.